Gruntfile.js

```diff
@@ -187,6 +205,7 @@ module.exports = function (grunt) {
         'mochaTest',
```

Comment:
While we're changing the grunt process, could we wait to lint until after running tests? It could save development time with Typescript already catching most errors.